Makefile

```diff
@@ -24,6 +24,9 @@ dependencies:
 	# We use 'go get' here because 'go install' will not build out of our vendored mock repo.
 	go get github.com/golang/mock/mockgen
 
+	# Necessary to generate thrift files, in addition to manually installing thrift via homebrew
+	go get github.com/apache/thrift/lib/go/thrift
```

Comment:
I'd hope they don't make that type of change lightly, but you're right, it's an uncontrolled variable. Actually, I'm not even sure why we'd need the thrift libraries to generate code.